### DISCUSSION TITLE
Is maxHttpBufferSize included in the python implementation of socket.io?

### DISCUSSION THREAD

**Discussion by marcom-db** [miguelgrinberg/python-socketio#787] [General]
In the node js implementation of socket.io there is an optional parameter to the server called maxHttpBufferSize: https://socket.io/docs/v3/server-initialization/#maxHttpBufferSize

Is this parameter included in this implementation?

**Comment by miguelgrinberg** [comment]
Yes. A simple check to the docs would have allowed you to answer your own question: https://python-socketio.readthedocs.io/en/latest/api.html#socketio.Server.
